script/Arcueid/Day 3/03_00_ARC03_3.txt

```diff
@@ -77,7 +77,7 @@ What magnanimity. I'm at a loss for words.
 [sha:26b0d26648653f5fd8a6ec4b78c544daa75612ef]{
 -- Page 132, Offset 4284.
 -- 「いや、それでも三年と二年じゃ距離が離れすぎです。ＨＲ、遅刻しても知りませんよ」
-"Alright, but the third- and second-year classrooms are pretty far apart.
+"Alright, but the third-year and second-year classrooms are pretty far apart.
  Don't blame me if you end up late for homeroom."
```

Comment:
you'd probably say it "third and second-year" if you were talking, so this should stay

maybe the dash on third isn't necessary but that's another question

Reply:
Im going with the former since I also dont think the hyphen is necessary